.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
I don't see pre-commit in the readme. Is this something people should install?

Reply:
`npx lint-staged`  : cf. [lint-staged](https://github.com/okonet/lint-staged) which is added as a dependency: https://github.com/smartcontractkit/hardhat-starter-kit/blob/fd8f7b8563bae016c00d0048e31f43c9bc88d2a4/package.json#L33

and pre-commits are managed by husky , which is also added as a dependency: 
https://github.com/smartcontractkit/hardhat-starter-kit/blob/fd8f7b8563bae016c00d0048e31f43c9bc88d2a4/package.json#L32